app/helpers/user_email_mailer_helper.rb

```diff
@@ -12,4 +12,42 @@ def btn_special(word, attr)
       'wrong attribute'
     end
   end
+
+  def random_position_and_style(category, word_count)
+    case category
+    when "positive"
+      color = "#4F9B64"
+    when "neutral"
+      color = "#4C77CB"
+    when "negative"
+      color = "#D1794E"
+    end
+
+    min_font_size = 12
+    max_font_size = 24
+    max_word_count = 3 # will be can adjust this value based on the desired number of repetitions of the word by the User
+    max_font_size_for_max_word_count = 28
+
+    if word_count >= max_word_count
+      font_size = max_font_size_for_max_word_count
+    else
+      random_factor = rand
+      font_size_range = max_font_size - min_font_size
+      font_size = min_font_size + (font_size_range * [word_count, max_word_count - 1].min.to_f / (max_word_count - 1) * random_factor)
+      font_size = font_size.round
+    end
+
+    font_families = [
+      'Arial', 'Helvetica', 'sans-serif', 'Times New Roman', 'Times', 'serif', 'Georgia',
+      'Palatino', 'Garamond', 'Bookman', 'Comic Sans MS', 'Trebuchet MS', 'Arial Black', 'Impact'
+    ]
+    font_family = font_families.sample
+
+    max_shift = font_size == max_font_size_for_max_word_count ? 5 : 28 - (font_size / 2).ceil
+
```

Comment:
It's not terribly clear what is going on here, which makes me think it should be in its own method with test coverage (which is a good way to explain to future developers what is going on).

Maybe instead of individual methods create a class to take care of both  this and calculating the font_size? Let me know if you'd like some help on explaining how to approach that.